net-im/nheko/nheko-0.11.3.recipe

```diff
@@ -0,0 +1,138 @@
+CHECKSUM_SHA256="f285156884a3a0c6870f3fba89c13d1fd70c8727bd179d8310b13819f8a63a37"
+SOURCE_FILENAME="nheko-${portVersion}.tar.gz"
+SOURCE_DIR="nheko-${portVersion}"
+PATCHES="nheko-${portVersion}.patchset"
+ADDITIONAL_FILES="nheko.rdef.in"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	nheko$secondaryArchSuffix = $portVersion
+	app:Nheko$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	gstreamer${secondaryArchSuffix}
+	noto_emoji
+	noto_emoji_color
+	dbus$secondaryArchSuffix
+	lib:libcurl$secondaryArchSuffix
+	lib:libre2$secondaryArchSuffix
+	lib:libgstwebrtc_1.0$secondaryArchSuffix
+	lib:libgstsdp_1.0$secondaryArchSuffix
+	lib:libglib_2.0$secondaryArchSuffix
+	lib:liborc_0.4$secondaryArchSuffix
+	lib:libcoeurl$secondaryArchSuffix
+	lib:libcmark$secondaryArchSuffix
+	lib:libcrypto$secondaryArchSuffix
+	lib:libfmt$secondaryArchSuffix
+	lib:liblmdb$secondaryArchSuffix
+	lib:libmatrix_client$secondaryArchSuffix
+	lib:libolm$secondaryArchSuffix
+	lib:libQt5Concurrent$secondaryArchSuffix
+	lib:libQt5Core$secondaryArchSuffix
+	lib:libQt5DBus$secondaryArchSuffix
+	lib:libQT5Gui$secondaryArchSuffix
+	lib:libqt5keychain$secondaryArchSuffix
+	lib:libQt5Multimedia$secondaryArchSuffix
+	lib:libQt5Qml$secondaryArchSuffix
+	lib:libQt5Quick$secondaryArchSuffix
+	lib:libQt5QuickControls2$secondaryArchSuffix
+	lib:libQt5QuickWidgets$secondaryArchSuffix
+	lib:libQt5Svg$secondaryArchSuffix
+	lib:libQt5Widgets$secondaryArchSuffix
+	lib:libspdlog$secondaryArchSuffix
+	lib:libssl$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	lmdbxx
+	asciidoc
+	extra_cmake_modules$secondaryArchSuffix >= 5.93
+	gstreamer${secondaryArchSuffix}_devel
+	dbus${secondaryArchSuffix}_devel
+	devel:libevent$secondaryArchSuffix
+	devel:libcurl$secondaryArchSuffix
+	devel:libre2$secondaryArchSuffix
+	devel:libgstwebrtc_1.0$secondaryArchSuffix
+	devel:libgstsdp_1.0$secondaryArchSuffix
+	devel:libglib_2.0$secondaryArchSuffix
+	devel:liborc_0.4$secondaryArchSuffix
+	devel:libcoeurl$secondaryArchSuffix
+	devel:libcmark$secondaryArchSuffix
+	devel:libfmt$secondaryArchSuffix
+	devel:liblmdb$secondaryArchSuffix
+	devel:libmatrix_client$secondaryArchSuffix  >= 0.5.1
+	devel:libolm$secondaryArchSuffix
+	devel:libQt5Concurrent$secondaryArchSuffix
+	devel:libQt5Core$secondaryArchSuffix
+	devel:libQt5Dbus$secondaryArchSuffix
+	devel:libQt5Gui$secondaryArchSuffix
+	devel:libqt5keychain$secondaryArchSuffix
+	devel:libQt5Multimedia$secondaryArchSuffix
+	devel:libQt5Network$secondaryArchSuffix
+	devel:libQt5Qml$secondaryArchSuffix
+	devel:libQt5Quick$secondaryArchSuffix
+	devel:libQt5QuickControls2$secondaryArchSuffix
+	devel:libQt5QuickWidgets$secondaryArchSuffix
+	devel:libQt5Svg$secondaryArchSuffix
+	devel:libQt5Widgets$secondaryArchSuffix
+	devel:libspdlog$secondaryArchSuffix
+	devel:libssl$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+	devel:nlohmann_json
```

Comment:
```suggestion
	devel:libcmark$secondaryArchSuffix
	devel:libcoeurl$secondaryArchSuffix
	devel:libcurl$secondaryArchSuffix
	devel:libevent$secondaryArchSuffix
	devel:libfmt$secondaryArchSuffix
	devel:libglib_2.0$secondaryArchSuffix
	devel:libgstsdp_1.0$secondaryArchSuffix
	devel:libgstwebrtc_1.0$secondaryArchSuffix
	devel:liblmdb$secondaryArchSuffix
	devel:libmatrix_client$secondaryArchSuffix  >= 0.5.1
	devel:libolm$secondaryArchSuffix
	devel:liborc_0.4$secondaryArchSuffix
	devel:libQt5Concurrent$secondaryArchSuffix
	devel:libQt5Core$secondaryArchSuffix
	devel:libQt5Dbus$secondaryArchSuffix
	devel:libQt5Gui$secondaryArchSuffix
	devel:libqt5keychain$secondaryArchSuffix
	devel:libQt5Multimedia$secondaryArchSuffix
	devel:libQt5Network$secondaryArchSuffix
	devel:libQt5Qml$secondaryArchSuffix
	devel:libQt5Quick$secondaryArchSuffix
	devel:libQt5QuickControls2$secondaryArchSuffix
	devel:libQt5QuickWidgets$secondaryArchSuffix
	devel:libQt5Svg$secondaryArchSuffix
	devel:libQt5Widgets$secondaryArchSuffix
	devel:libre2$secondaryArchSuffix
	devel:libspdlog$secondaryArchSuffix
	devel:libssl$secondaryArchSuffix
	devel:libz$secondaryArchSuffix
	devel:nlohmann_json
```

Reply:
is devel:libevent actually needed?

Reply:
Yes, without it has a build error "No package 'libevent_core' found"

Reply:
OK Thanks!